tests/core/editable/insertion.js

```diff
@@ -4,7 +4,8 @@
 	'use strict';
 
 	var doc = CKEDITOR.document,
-		tools = bender.tools;
+		tools = bender.tools,
+		space = CKEDITOR.env.ie ? ' ' : '&nbsp;';
```

Comment:
You are declaring space here for IE issue, but at the same time IE tests are ignored and the test is working on Edge without this modification.

Reply:
I think this value should be here, as this is used for the IE, however because of other bug test is ignored. Once #3061 is resolved test can be just unignored.